### PR TITLE
feat(haveged): add haveged

### PR DESCRIPTION
**Pull request by philippfranke**

### CODE REVIEW

File: images/haveged/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM arm32v6/alpine:3.7
```
Comment (gq0) [inline images/haveged/Dockerfile]: Actually we have a armv7 but could you use 
```FROM multiarch/debian-debootstrap:armhf-stretch-slim```

**Review by gq0 — CHANGES_REQUESTED** [review]

File: images/haveged/Dockerfile (hunk 1 above)
Comment (gq0) [reply images/haveged/Dockerfile]: to make the system aware of arm binaries and to use qemu:
`docker run --rm --privileged multiarch/qemu-user-static:register --reset`

**Comment by jhedev** [comment]
@philippfranke are you going to fix this?

**Comment by jhedev** [comment]
@philippfranke are you going to update this or should we close it?

**Comment by jhedev** [comment]
you updated it already :D sorry. Zube doesn't show that.

File: images/haveged/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM multiarch/debian-debootstrap:armhf-stretch-slim
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+				haveged && \
+	rm -rf /var/lib/apt/lists/*
```
Comment (jhedev) [inline images/haveged/Dockerfile]: weird formatting. pls fix it.

**Review by jhedev — CHANGES_REQUESTED** [review]

**Comment by jhedev** [comment]
@gq0 please have a look again

File: images/haveged/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+ENTRYPOINT ["/usr/sbin/haveged", "-F"]
```
Comment (gq0) [inline images/haveged/Dockerfile]: I am not sure what the default number of bits but maybe add `-w 1024` to be sure.

**Review by jhedev — APPROVED** [review]

**Review by gq0 — APPROVED** [review]

**Pull request merged**